bundles/org.eclipse.swt/Eclipse SWT Browser/win32/org/eclipse/swt/browser/Edge.java

```diff
@@ -549,7 +575,11 @@ public String getText() {
 public String getUrl() {
 	long ppsz[] = new long[1];
 	webView.get_Source(ppsz);
-	return wstrToString(ppsz[0], true);
+	return getExposedUrl(wstrToString(ppsz[0], true));
+}
+
+private String getExposedUrl(String url) {
+	return isLocationForCustomText(url) ? url = ABOUT_BLANK : url;
```

Comment:
```suggestion
	return isLocationForCustomText(url) ? ABOUT_BLANK : url;
```
(no need to also assign url)

Reply:
Done